src/GraphQL.Tests/Bugs/Bug256NullableEnumTests.cs

```diff
@@ -29,7 +30,7 @@ public void nullable_enum_returns_null()
         {
             var ctx = new ResolveFieldContext
             {
-                Arguments = new Dictionary<string, object> { { "value", null } }
+                Arguments = new Dictionary<string, ArgumentValue> { { "value", new ArgumentValue(null, ArgumentSource.Literal) } }
```

Comment:
```c#
public readonly struct ArgumentValue
{
    public static readonly ArgumentValue NullLiteral = new ArgumentValue(null, ArgumentSource.Literal);
    public static readonly ArgumentValue NullVariable = new ArgumentValue(null, ArgumentSource.Variable);
}
```

`Arguments = new Dictionary<string, ArgumentValue> { { "value", new ArgumentValue(null, ArgumentSource.Literal) } }`
->
`   Arguments = new Dictionary<string, ArgumentValue> { ["value"] = ArgumentValue.NullLiteral }`

Reply:
Looks fine, but I'm not going to go change all the tests because the code looks prettier.  It took me an hour to fix all the tests the first time!  I'll add the variables into the struct if you like.

Reply:
I'll do a search-and-replace and catch whatever that turns up.

Reply:
Well, if there is a way to make the code no worse than it was, then why not?. At least for simple int/string values.

Reply:
I changed them to readonly properties so they do not cause a CPU cache miss.  It should be equally as fast otherwise.

Reply:
Of course. I wrote fields out of habit.